ext/openssl/openssl.c

```diff
@@ -705,6 +705,12 @@ static int X509_get_signature_nid(const X509 *x)
 
 #endif
 
+#define OpenSSL_version		SSLeay_version
```

Comment:
Doesn't this cause double definitions for configuration without `OPENSSL_API_COMPAT` (e.g. for config like: `./config shared`?

Reply:
I believe this code block is found within an ```#if``` directive. https://github.com/php/php-src/blob/master/ext/openssl/openssl.c#L582